go.mod

```diff
@@ -1,36 +1,46 @@
 module github.com/krakendio/krakend-lua/v2
 
-go 1.17
+go 1.19
 
 require (
-	github.com/gin-gonic/gin v1.7.7
+	github.com/gin-gonic/gin v1.10.0
```

Comment:
we'll keep on 1.9.1 until we decide what to do with the change in the returned content-type headers